app/services/cfe/regular_transactions_payload_service.rb

```diff
@@ -7,7 +7,7 @@ def call
         relevant_form?(:housing_costs) ||
         relevant_form?(:benefit_details)
 
-      outgoings_form = instantiate_form(OutgoingsForm)
+      outgoings_form = instantiate_form(OutgoingsForm) if relevant_form?(:outgoings)
       income_form = instantiate_form(OtherIncomeForm)
```

Comment:
do we need to check the form is relevant?

Reply:
Stephen has implemented this change here: https://github.com/ministryofjustice/laa-check-client-qualifies/pull/1261

so won't do on this PR